packages/EasyWebhook/src/Stores/DoctrineDbalResultStore.php

```diff
@@ -14,18 +14,27 @@
 
 final class DoctrineDbalResultStore extends AbstractDoctrineDbalStore implements ResultStoreInterface
 {
+    /**
+     * @var string|null
+     */
+    private $timestampTimezone;
+
     public function __construct(
         RandomGeneratorInterface $random,
         Connection $conn,
         DataCleanerInterface $dataCleaner,
-        ?string $table = null
+        ?string $table = null,
+        ?string $timestampTimezone = null
     ) {
+        $this->timestampTimezone = $timestampTimezone;
         parent::__construct($random, $conn, $dataCleaner, $table ?? 'easy_webhook_results');
     }
 
     public function store(WebhookResultInterface $result): WebhookResultInterface
     {
-        $now = Carbon::now('UTC');
+        $timezone = $this->timestampTimezone ?? 'UTC';
```

Comment:
Let's rename timestampTimezone to "timezone". Let's be specific. timestamp refers to a different concept.

Also let's add a getter method

Reply:
@evannamon agree with @josemujicap please let's rename it to simply `timezone`

Reply:
@josemujicap & @natepage fixed variable, added getter and assertion for the added method.